neural_modelling/src/neuron/plasticity/stdp/synapse_dynamics_stdp_mad_impl.c

```diff
@@ -92,7 +93,8 @@ static inline final_state_t _plasticity_update_synapse(
     // Process events in post-synaptic window
     while (post_window.num_events > 0) {
         const uint32_t delayed_post_time = *post_window.next_time
-                                           + delay_dendritic;
+//                                           + delay_dendritic;
+                                           + 0;
```

Comment:
Setting a spike backpropagation delay of `delay_dendritic` is an issue that affects autapses; they are potentiated instead of being depressed. The change can be safely ignored and reverted to the original `+ delay_dendritic;`, however, in future, we will need to model axonal and dendritic delay differently. @rowleya & @oliverrhodes , do you guys have anything to add?

Reply:
For now, I would say revert this change and add a ticket (if there isn't one already).